h/collections.h

```diff
@@ -307,8 +315,23 @@ void qInsert(Queue *q, void *newData) QueueAmode64;
  */
 void *qRemove(Queue *q) QueueAmode64;
 
-#endif
+typedef struct ArrayList_tag{
+  int capacity;
```

Comment:
Perhaps `size_t` to support sizes above 4Gb?

Reply:
Maybe someday.   I am scared of linear 2GB data structures.   I hope something else would be used for such a case.